python/lsst/meas/algorithms/convertRefcatManager.py

```diff
@@ -454,3 +454,35 @@ def gaiaFluxToFlux(flux, zeroPoint):
         result['phot_rp_mean_fluxErr'] = result['phot_rp_mean_flux'] / input['phot_rp_mean_flux_over_error']
 
         return result
+
+class ConvertGaiaXpManager(ConvertRefcatManager):
+    """Special-case convert manager to deal with refcats that have no MagErrors
```

Comment:
Maybe: "Special-case convert manager for Gaia XP spectrophotometry catalogs, that have fluxes/flux errors, instead of magnitudes."